### PR TITLE
Fix warn

### LINKED ISSUE #765: Warn not showing up for the ops
I've been using /warn for minor offenses lately, the warning shows up for the admin, not the op. This is a bug that I hope gets fixed. Let me look over the command.

### PR DESCRIPTION
**Pull request by MrImmaculate**
Currently sends the warning only to the admin, so this fixes it. I don't think any other method is necessary, this causes no conflicts.

Closes #765

### CODE REVIEW

**Comment by DUFCLiam** [comment]
@Chaosiity, Please create an issue

**Comment by Chaosiity** [comment]
@DUFCLiam There was already an accepted issue, I will see if I can find it.

**Comment by DUFCLiam** [comment]
Ok :)

**Comment by ImScoop** [comment]
There is no current issue regarding this PR. I will make one.

**Comment by DUFCLiam** [comment]
@ImScoop, Please change player.sendMessage(string); to playerMsg(string);

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_warn.java [hunk 1]
```diff
@@ -46,7 +46,8 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
 
         String warnReason = StringUtils.join(ArrayUtils.subarray(args, 1, args.length), " ");
 
-        playerMsg(ChatColor.RED + "[WARNING] " + warnReason);
+        player.sendMessage(ChatColor.RED + "[WARNING] " + ChatColor.YELLOW + warnReason);
```
Comment (DUFCLiam) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_warn.java]: playerMsg(ChatColor.RED + "[WARNING] " + ChatColor.YELLOW + warnReason);

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_warn.java [hunk 2]
```diff
@@ -46,7 +46,8 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
...
+        player.sendMessage(ChatColor.RED + "Warned by " + sender.getName());
```
Comment (DUFCLiam) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_warn.java]: playerMsg(ChatColor.RED + "Warned by " + sender.getName());

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_warn.java (hunk 1 above)
Comment (AvalancheYT) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_warn.java]: No @DUFCLiam, playerMsg only send to the command sender

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_warn.java (hunk 2 above)
Comment (AvalancheYT) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_warn.java]: No @DUFCLiam, playerMsg only send to the Command sender

Comment (DUFCLiam) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_warn.java]: Ahh

**Comment by ImScoop** [comment]
@JeromSar this closes an issue, please review and merge.

**Comment by Wild1145** [comment]
I'd like to have it as one line really, something like "[WARNING] XYZ has banned you for [Reason]"

I think that would be cleaner.

**Comment by ImScoop** [comment]
@Wild1145 Up to @JeromSar.

**Comment by MrImmaculate** [comment]
@JeromSar What further review does this need? Haha.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_warn.java (hunk 1 above)
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_warn.java]: `playerMsg()` can take a second argument, the recepient, I believe.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_warn.java (hunk 2 above)
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_warn.java]: Same as above.

**Comment by MrImmaculate** [comment]
Cannot be edited. Opening new one with requested changes.

**Pull request closed without merging**